frontend-react/src/styles/_reportstream.scss

```diff
@@ -795,3 +795,49 @@ pre {
         margin-top: 40px;
     }
 }
+
```

Comment:
I think this CSS should be within the `_components.scss` file.

Reply:
@jpandersen87 are you in alignment with moving these styles?

Reply:
👍